platform/backend/src/standalone-scripts/codegen-access-control-docs.ts

```diff
@@ -24,8 +24,7 @@ function getResourceDescription(resource: Resource): string {
     interaction: "Conversation history and agent interactions",
     dualLlmConfig: "Dual LLM security configuration settings",
     dualLlmResult: "Results from dual LLM security validation",
-    settings: "Organization-wide configuration settings",
```

Comment:
`settings` was defined as a valid `Resource` but it's not actually used anywhere, removing it so as to simplify access control documentation